nuxeo-runtime/nuxeo-runtime-deploy/src/test/java/org/nuxeo/runtime/deployment/preprocessor/install/commands/AppendCommandTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * (C) Copyright 2016 Nuxeo SA (http://nuxeo.com/) and contributors.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the GNU Lesser General Public License
+ * (LGPL) version 2.1 which accompanies this distribution, and is available at
+ * http://www.gnu.org/licenses/lgpl-2.1.html
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * Contributors:
+ *      Nelson Silva <nsilva@nuxeo.com>
+ */
+package org.nuxeo.runtime.deployment.preprocessor.install.commands;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.nuxeo.common.utils.FileUtils;
+import org.nuxeo.common.utils.Path;
+import org.nuxeo.runtime.deployment.preprocessor.install.Command;
+import org.nuxeo.runtime.deployment.preprocessor.install.CommandContext;
+import org.nuxeo.runtime.deployment.preprocessor.install.CommandContextImpl;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @since 8.4
+ */
+public class AppendCommandTest {
+
+    @Rule
+    public TemporaryFolder tmp = new TemporaryFolder();
+
+    private CommandContext ctx;
+
+    private ObjectMapper om;
+
+    @Before
+    public void setUp() {
+        ctx = new CommandContextImpl(tmp.getRoot());
+        om = new ObjectMapper();
+    }
+
+    @Test(expected = IOException.class)
```

Comment:
We prefer an explicit:

    try {
        ...stuff...
        fail("Should throw IOException");
    } catch (IOException e) {
        assertEquals("xxx", e.getMessage());
    }